enhancements/single-node/aos-sno-pair.md

```diff
@@ -0,0 +1,143 @@
+---
+title: aos-sno-pair
+authors:
+- @mshitrit
+  reviewers:
+- TBD
+  approvers:
+- TBD
+
+creation-date: 2021-07-12
+last-updated: 2021-07-12
+status: implementable
+---
+
+# Atomic OpenShift Single-Node-Pair
+
+## Release Signoff Checklist
+
+- [x] Enhancement is `implementable`
+- [ ] Design details are appropriately documented from clear requirements
+- [ ] Test plan is defined
+- [ ] Graduation criteria for dev preview, tech preview, GA
+- [ ] User-facing documentation is created in [openshift-docs](https://github.com/openshift/openshift-docs/)
+
+## Summary
+This enhancement describes setting up a two nodes cluster with a reduced footprint which will serve edge cases.
+
+## Motivation
+
+Some companies have a need for a highly available container management solution that fits within a reduced footprint.
+- The hardware savings are significant for customers deploying many remote sites (kiosks, branch offices, restaurant chains, etc), most notably for edge computing and RAN specifically.
+- The physical constraints of some deployments prevents more than two nodes (planes, submarines, satellites, and also RAN).
+- Some locations will not have reliable network connections or limited bandwidth (once again submarines, satellites, and disaster areas such as after hurricanes or floods)
+  
+### Goals
+
+Provide the ability for all-in-one OpenShift clusters to operate as a **predefined** pair in an **active-active** configuration, detect when its peer has died and **automatically** take over its workloads after ensuring it is safe to do so.
```

Comment:
Is this the goal? Based on what has been said elsewhere, the clusters aren't actually aware of each other, it's the applications running on them that are paired. Did I misunderstand those other comments?

Reply:
I think that you understood correctly the part that indeed each SNO is independent.
However both SNO + HA Stack compromise the all-in-one OpenShift cluster (AOS) which is another entity.
The above applies to AOS entity and not to the SNOs.

Reply:
I'm not sure what you mean by "compromise".

Reply:
Sorry typo - I meant compose

Reply:
Ah, ok, that makes more sense. I think my comments in the other thread about platform vs. application apply here, too. Let's carry on that conversation in the other thread so it's all in one place.

Reply:
```suggestion
Provide the ability for exactly two all-in-one OpenShift clusters to operate as a **predefined** pair in an **active-active** configuration, detect when its peer has died and **automatically** take over its workloads after ensuring it is safe to do so.
```